PatternPal/PatternPal.LoggingServer/Protos/log.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+option csharp_namespace = "PatternPal.LoggingServer";
+
+package log;
+
+// The greeting service definition.
+service Log {
```

Comment:
In the other gRPC services, I've used the convention of postfixing the service with `Service`, I'd like to stick to that convention.

It's probably also a good idea to be clear about the difference between the logging service which is exposed by the background process, and the logging service which runs remotely and collects the data. Perhaps this should be `LogCollectorService`, and the service exposed by the background process could simply be `LogService`.

```suggestion
service LogCollectorService {
```